modules/install-sno-about-single-node-openshift.adoc

```diff
@@ -0,0 +1,10 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-preparing-to-install-sno.adoc
+
+[id="install-sno-about-single-node-openshift_{context}"]
+= About single node OpenShift
+
+Single node OpenShift installs a self-contained {product-title} instance on a single bare metal node. The primary use case is for edge computing workloads, including small physical footprints, intermittent connectivity, portable clouds, and 5G radio access networks (RAN) close to a base station. A single node {product-title} installation supports autonomous workloads.
```

Comment:
According to the convo is Slack, this is OCP, just a specialized installation on bare metal, like a three node cluster.

Perhaps "An installation of {product-title} on a single node is a specialized installation. The primary use case..."  I don't know how critical it is to include "self-contained."

For the last sentence, please use "An installation on a single node supports autonomous workloads." or "An {product-title} installation on a single node supports..."  My interest is to separate "single node" from the product name so that customers do not interpret this as a new product.